components/match2/commons/match_group.lua

```diff
@@ -21,6 +23,8 @@ local MatchGroupUtil = Lua.import('Module:MatchGroup/Util', {requireDevIfEnabled
 local ShortenBracket = Lua.import('Module:MatchGroup/ShortenBracket', {requireDevIfEnabled = true})
 local WikiSpecific = Lua.import('Module:Brkts/WikiSpecific', {requireDevIfEnabled = true})
 
+local MATCHLIST_MATCH_GROUP_TYPE = 'matchlist'
```

Comment:
Considering the amount of places `matchlist` and `bracket` are hardcoded, keep it like that. If moved to a static, then it should be in a common place, so only need to update it at one place, not in every single module